### PR TITLE
duplicity 3.0.0

### PR DESCRIPTION
**Pull request by branchv**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

---

### CODE REVIEW

File: Formula/d/duplicity.rb [hunk 1]
```diff
@@ -285,8 +325,8 @@ class Duplicity < Formula
   end
 
   resource "pbr" do
-    url "https://files.pythonhosted.org/packages/8d/c2/ee43b3b11bf2b40e56536183fc9f22afbb04e882720332b6276ee2454c24/pbr-6.0.0.tar.gz"
-    sha256 "d1377122a5a00e2f940ee482999518efe16d745d423a670c27773dfbc3c9a7d9"
+    url "https://files.pythonhosted.org/packages/c3/2c/63275fab26a0fd8cadafca71a3623e4d0f0ee8ed7124a5bb128853d178a7/pbr-1.10.0.tar.gz"
+    sha256 "186428c270309e6fdfe2d5ab0949ab21ae5f7dea831eab96701b86bd666af39c"
```
Comment (chenrui333) [inline Formula/d/duplicity.rb:329]: this needs to be reverted

Comment (branchvincent) [reply Formula/d/duplicity.rb:329]: yea a bunch of versions got downgraded, not sure why

Comment (branchvincent) [reply Formula/d/duplicity.rb:329]: This is an upstream bug (I don't have a gitlab account to report myself), they are now properly declaring their dependencies but are all pinned and some to very old versions: https://gitlab.com/duplicity/duplicity/-/blob/dev/requirements.txt

Comment (chenrui333) [reply Formula/d/duplicity.rb:329]: :(

**Comment by chenrui333** [comment]
there is 3.0.0 release now

**Comment by chenrui333** [comment]
`  ModuleNotFoundError: No module named 'urllib3.packages.six.moves'`

**Comment by branchvincent** [comment]
Forced a newer `urllib3`, they have it pinned to a version that doesn't work on 3.12

**Review by chenrui333 — APPROVED** [review]
💯

**Review by chenrui333 — APPROVED** [review]

**Pull request merged**